packages/testlab/src/request.ts

```diff
@@ -3,29 +3,39 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {IncomingMessage} from 'http';
 import * as http from 'http';
+import {IncomingMessage} from 'http';
 import * as https from 'https';
 import * as url from 'url';
 
 /**
  * Async wrapper for making HTTP GET requests
  * @param urlString
  */
-export function httpGetAsync(urlString: string): Promise<IncomingMessage> {
+export function httpGetAsync(
+  urlString: string,
+  agent?: http.Agent,
+): Promise<IncomingMessage> {
```

Comment:
These testlab changes would be best to make in a standalone commit, so that we get a meaningful changelog entry.

Reply:
The change to `testlab` is already in its own commit.